src/types/modules/react.d.ts

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+declare module 'react' {
+    // eslint-disable-next-line @typescript-eslint/ban-types
+    function forwardRef<T, P = {}>(render: (props: P, ref: React.Ref<T>) => React.ReactElement | null): (props: P & React.RefAttributes<T>) => React.ReactElement | null;
```

Comment:
Thanks for testing it guys! 

Possible tiny improvement with using `ForwardedRef` instead of regular `Ref`

```typescript
function forwardRef<T, P = {}>(render: (props: P, ref: React.ForwardedRef<T>) => React.ReactElement | null): (props: P & React.RefAttributes<T>) => React.ReactElement | null;
```

Reply:
Done! :D 
Thanks!